setup.py

```diff
@@ -62,7 +62,7 @@ def get_version(rel_path: str) -> str:
     "matplotlib>=3.3",
     "tables>=3.6.1",
     "pyyaml>=5.3.1",
-    "mlflow>=1.12.1",
+    "mlflow<=1.30.0",
     "tqdm",
```

Comment:
you might consider this way to force the mlflow min vresion.
"mlflow>=1.12.1, <=1.30.0",